src/components/styles.scss

```
@@ -206,7 +206,9 @@ select {
       }
     }
   }
-  
+  img {
```

As it stands, if we hardcode an image inside of that `.primary-content` div later it will have `max-width: 100%` whether we want it to or not. Since it looks like all of the `<img>`s that are provided via markdown are wrapped in a `<p>`, we can be more specific about _which_ images we want to apply this style too.

```suggestion
  p img {
```